Packs/DevSecOps/Playbooks/DevSecOps_-_Prisma_-_Analysis_-SubPB.yml

```diff
@@ -0,0 +1,320 @@
+      stixtooltypes: {}
+      stixtoolversion: {}
+      subdomains: {}
+      tags: {}
+      threattypes: {}
+      trafficlightprotocol: {}
+      type:
+        simple: IaC Vulnerability
+      updateddate: {}
+      username: {}
+      value:
+        complex:
+          root: inputs.Prisma_Cloud_Check_Run_Summary
+          transformers:
+          - operator: substringFrom
+            args:
+              from:
+                value:
+                  simple: '------------ | ------------- | ------------ | -------------'
+          - operator: substringTo
+            args:
+              to:
+                value:
+                  simple: '|'
+      vulnerabilityname: {}
+      xdrstatus: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: 64f0586c-08c6-4f20-80ee-49c3bb594c12
+    type: title
+    task:
+      id: 64f0586c-08c6-4f20-80ee-49c3bb594c12
+      version: -1
+      name: Playbook End
+      description: commands.local.cmd.close.inv
+      type: title
+      iscommand: false
+      brand: Builtin
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: 83ec245b-dad6-40aa-8dde-2d6c2143279a
+    type: condition
+    task:
+      id: 83ec245b-dad6-40aa-8dde-2d6c2143279a
+      version: -1
+      name: IaC Vulns Found?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: startWith
+          left:
+            value:
+              simple: inputs.Prisma_Cloud_Check_Run_Title
+            iscontext: true
+          right:
+            value:
+              simple: Vulnerabilities Found
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 560,
+        "width": 492.5,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: Prisma_Cloud_Check_Run_Summary
+  value: {}
+  required: false
+  description: Prisma Cloud Check Run Summary
+  playbookInputQuery:
+- key: Prisma_Cloud_Check_Run_Title
+  value: {}
+  required: false
+  description: Prisma Cloud Check Run Title
+  playbookInputQuery:
+outputs: []
+tests:
+- GitLab-test-playbook
+- LGTM-test-playbook
+- playbook-MinIO-Test
+fromversion: 1.0.0
```

Comment:
```suggestion
fromversion: 5.0.0
```
The version should be consistent across all files in the Pack
(It's currently 5.0.0) Please fix this.